tests/test_cli.py

```diff
@@ -62,6 +62,52 @@ def test_show__format_html():
     assert result.exit_code == ExitCodes.OK
 
 
+def test_show__format_json():
+    from pycobertura.cli import show, ExitCodes
+
+    runner = CliRunner()
+    for opt in ('-f', '--format'):
+        result = runner.invoke(
+            show,
+            ['tests/dummy.original.xml', opt, 'json'],
+            catch_exceptions=False
+        )
+        assert result.output == """\
+{
+    "total": {
+        "Filename": "TOTAL",
+        "Stmts": 4,
+        "Miss": 2,
+        "Cover": "50.00%"
+    },
+    "files": [
```

Comment:
I see `files` in an array, is it necessary? Will we ever have an array of more than 1 item?

Reply:
done